public/cli/lib/commands/timing.js

```diff
@@ -131,6 +134,10 @@ function checkInputExists(done) {
             return;
         }
         inputData = contents;
+
+        // since that worked, lets also scan this dir for all audio files
```

Comment:
Strictly speaking this changes what the `checkInputExists` method does. Either change the method name or put this functionality in its own method.

Reply:
ok fffffiiinnnnneeeee : I've fixed the n00b "just throw in some code that doesn't follow the function intention" error.

Reply:
Thanks.